src/Containers.scss

```diff
@@ -33,3 +33,18 @@
 .ct-table-empty td {
     padding: var(--pf-global--spacer--sm) var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
 }
+
+// Fix label color and alignment in the table cell
+.container-state {
+  // Unless there is a modifier (ex: pf-m-green for running), drop the color
+  &:not([class*=pf-m-]) {
+    --pf-c-label--BackgroundColor: transparent;
+    padding-right: 0;
+    // Compensate for padding
+    position: relative;
+    // Using margin to remove the extra padded space,
+    // instead of keeping the space and moving contents w/ just left/top
+    margin-left: calc(-1 * var(--pf-c-label--PaddingLeft));
```

Comment:
With this the mobile mode looks nice but on regular monitor sizes it looks worse:
Without this line:

![Screen Shot 2021-02-04 at 12 17 44](https://user-images.githubusercontent.com/14921356/106885668-3b187500-66e3-11eb-9d2c-e9f7102fdb4b.png)

With this line:

![Screen Shot 2021-02-04 at 12 17 16](https://user-images.githubusercontent.com/14921356/106885672-3b187500-66e3-11eb-97c4-a91d5a763364.png)

Reply:
I had something to offset the state for the running label too, but I don't see it here in your screenshots. Perhaps it got lost in a rebase.